packages/core/src/components/Button/Button.jsx

```diff
@@ -136,7 +139,7 @@ Button.propTypes = {
    */
   onClick: PropTypes.func,
   /**
-   * (Deprecated) Access a reference to the `button` or `a` element. Please use `inputRef` instead.
+   * @HidePropDoc [Deprecated] Access a reference to the `button` or `a` element. Please use `inputRef` instead.
```

Comment:
I don't think `@HidePropDoc` is working as expected. I saw this in the button props section.
![Screen Shot 2020-02-17 at 9 48 26 AM](https://user-images.githubusercontent.com/1449852/74663971-d5964600-516a-11ea-8874-5e2dc6ac727c.png)